crates/solidity/src/solc/solc_compiler.rs

```diff
@@ -1,16 +1,24 @@
 //! The Solidity compiler.
 
-use std::io::Write;
-use std::path::Path;
-use std::path::PathBuf;
-
 use crate::solc::combined_json::CombinedJson;
 use crate::solc::pipeline::Pipeline;
 use crate::solc::standard_json::input::Input as StandardJsonInput;
 use crate::solc::standard_json::output::Output as StandardJsonOutput;
 use crate::solc::version::Version;
+use once_cell::sync::Lazy;
+use semver::VersionReq;
+use std::io::Write;
+use std::path::Path;
+use std::path::PathBuf;
 
 use super::Compiler;
+// `--base-path` was introduced in 0.6.9 <https://github.com/ethereum/solidity/releases/tag/v0.6.9>
+pub static SUPPORTS_BASE_PATH: Lazy<VersionReq> =
+    Lazy::new(|| VersionReq::parse(">=0.6.9").unwrap());
+
+// `--include-path` was introduced in 0.8.8 <https://github.com/ethereum/solidity/releases/tag/v0.8.8>
+pub static SUPPORTS_INCLUDE_PATH: Lazy<VersionReq> =
+    Lazy::new(|| VersionReq::parse(">=0.8.8").unwrap());
```

Comment:
Please move those next to the other solc version defined constants in `crates/solidity/src/solc/mod.rs`.

Please also rename to follow the existing naming (`FIRST_BASE_PATH_VERSION`  and `FIRST_INCLUDE_PATH_VERSION`)